mnefun/_report.py

```diff
@@ -335,22 +337,24 @@ def gen_html_report(p, subjects, structurals, run_indices=None,
                     stc = abs(stc)
                     # get clim using the reject_tmin <->reject_tmax
                     stc_crop = stc.copy().crop(p.reject_tmin, p.reject_tmax)
-                    colormap = source.get('colormap', 'viridis')
-                    transparent = source.get('transparent', True)
```

Comment:
What sort of `dict` magic is this?
I don't see where a lot of the `source` `(key, value)' pairs are set e.g., `('transparent', True)`, and what happens with `dict.get(key, value)` statements?

Reply:
https://docs.python.org/2/library/stdtypes.html#dict.get